Libraries/D3D12RaytracingFallback/developerguide.md

```diff
@@ -209,14 +209,17 @@ The Fallback Layer natively works with PIX. However, PIX does not have support f
 When using the debug version of the Fallback Layer library, the Fallback layer has limited validation that will catch unsupported cases and output the cause of failure to the debugger. 
 
 ### Acceleration structure visualization
-It can be helpful to visualize the acceleration structure, particularly when you make changes to the structure logic in the library. To do this, enable ENABLE_ACCELERATION_STRUCTURE_VISUALIZATION in [FallbackDebug.h](src/FallbackDebug.h) and set FallbackLayer::m_levelToVisualize in [FallbackLayer.h](src/FallbackLayer.h) to a tree level you want to visualize.
+It can be helpful to visualize the acceleration structure, particularly when you make changes to the structure logic in the library. To do this, enable ENABLE_ACCELERATION_STRUCTURE_VISUALIZATION in [FallbackDebug.h](src/FallbackDebug.h) and in [ModelViewerRayTracing.h](../../Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingMiniEngineSample/ModelViewerRayTracing.h).
```

Comment:
Does it really need to be defined in two places?

Reply:
....yyyes?

I think?